frontend/src/metabase/lib/analytics.js

```diff
@@ -20,7 +20,14 @@ export const trackPageView = url => {
   }
 };
 
-export const trackSchemaEvent = (schema, version, data) => {
+export const trackSchemaEvent = (schema, version, data, contextEntities) => {
+  /*
+  IMPORTANT NOTE:
+  Only use this function directly if `trackActionEvent` doesn't fit your needs.
+  We want to use the generic structured action event as much as possible to avoid to create or update schemas.
+  If you need to track event specific context that does not fit the structured action event, you can pass context entities
+  that are designed in a reusable way (https://docs.snowplow.io/docs/understanding-your-pipeline/entities/)
+  */
```

Comment:
This would be nice as a JSDoc so that we can have the necessary information in auto-complete and other intelli-sense prompts.

Reply:
Adjusted :)